java.header

```diff
@@ -1,15 +1,29 @@
 ^/\*$
-^ \* Copyright \d\d\d\d,? Google (Inc\.|LLC)( All [rR]ights [rR]eserved\.)?$
+^ \* Copyright 20\d\d Google LLC$
```

Comment:
What's the reason for the copyright change?

Reply:
Reverting.